common/djangoapps/util/tests/test_submit_feedback.py

```diff
@@ -184,6 +198,9 @@ def _build_zendesk_ticket_update(self, request_headers, username=None):
             ("Browser", "HTTP_USER_AGENT")
         ]
 
+        if getattr(settings, 'ENABLE_ENTERPRISE_INTEGRATION', False):
+            header_text_mapping.insert(0, ("enterprise_customer_name", "ENTERPRISE_CUSTOMER_NAME"))
```

Comment:
When integration is off then I cannot expect the `enterprise_customer_name` in header text.